website/tutorial/managing-your-application-resources.md

```diff
@@ -0,0 +1,198 @@
+
+5. Access other useful actions:
+   - Use the **Prune** button to remove all unused images from the Podman engine.
+   - Load images from a tar archive on your local machine.
+   - Import images from your local machine into the Podman Desktop application.
+   - Pull an image from a configured registry.
+   - Build an image from a container or docker file.
+
+## Managing volumes
+
+Perform any of the following actions after navigating to the component page.
+
+![volume actions](img/volume-actions.png)
+
+1. Search a volume using the search box.
+2. Use the **Delete** icon to delete a volume. You can also select multiple volumes and delete them in one step by using the delete button on the page.
+
+   :::note
+
+   - You cannot delete a volume if it is being used in any containers. In such cases, you must delete the container and then delete the volume.
+   - By selecting multiple volumes, you can delete them in a single step.
+
+   :::
+
+3. Click the name of the volume to view its summary.
+4. Access other useful actions:
+
+- Use the **Prune** button to remove unused volumes from the Podman engine.
+- Click **Gather volume sizes** to determine the size used for volumes.
+- Click the **Create** button to create a volume.
+
+## Managing Kubernetes objects
+
+Perform any of the following actions after navigating to the component page.
+
+![object actions](img/object-actions.png)
+
+1. Search for objects using the search box.
+2. Use the **Delete** icon to delete an object.
+3. Use the **Apply YAML** button to create an object.
+4. Click the name of the object to:
+   - View its summary
+   - View its Kubernetes configuration
+   - Edit its Kubernetes configuration and apply those changes to your cluster.
+
+## Managing other resources
+
+You can manage other resources, such as your Podman machine, cluster, or CLI tools using the Podman Desktop Settings.
+
+**_Managing your Podman machine_**
+
+On the **Settings > Resources** page, perform any of the following actions:
+
+![podman machine set up](img/podman-setup.png)
+
+1. Create a new Podman machine.
+2. Set up Podman machine preferences.
+3. Access other useful actions to start, restart, stop, edit, or delete a Podman machine.
+
+**_Managing your cluster_**
+
+On the **Settings > Resources** page, perform any of the following actions:
+
+![cluster set up](img/cluster-setup.png)
+
+1. Create a new cluster.
+2. Access other useful actions to start, restart, stop, or delete a cluster.
+
+**_Managing your CLI tools_**
+
+If you have installed a CLI tool extension, you can set up the CLI tool and upgrade to its latest version directly from the UI.
+
+- On the **Settings > Resources** page, click **Setup** to download the CLI tool and install it system wide.
+
+![CLI set up](img/set-up-cli.png)
+
+- On the **Settings > CLI tools** page, click the updates available icon to upgrade to the latest version of the CLI tool:
+
+![CLI update](img/cli-update.png)
```

Comment:
Maybe we should remove the `kind` area of this screenshot. Just to make it a bit more compact. Since the `Upgrade` button is in the `docker-compose` area